### PR TITLE
fix: s3archive response

### PR DESCRIPTION
**Pull request by marcellmueller**
Eslint was complaining that I didn't return anything from async function so I threw a `return res.end()` which prematurely ended the function. Also added missing filenames.

### CODE REVIEW

File: app/backend/lib/s3archive.ts [hunk 1]
```diff
@@ -95,9 +95,7 @@ s3archive.get('/api/analyst/archive', async (req, res) => {
     sortAndAppendAttachments(jsonData, ccbcId);
   });
 
-  archive.finalize();
```
Comment (AntBush) [inline app/backend/lib/s3archive.ts:98]: Would adding an `await` here also have solved the problem? I assume it was returning before this had finished.

**Review by AntBush — APPROVED** [review]
Looks good!

Unrelated to this, if we want to be less CPU intensive, we can set the archiver option `{ store: true }` to turn off compression

File: app/backend/lib/s3archive.ts (hunk 1 above)
Comment (marcellmueller) [reply app/backend/lib/s3archive.ts:98]: I did try that but it would still return the res.end before the archive finalized.

**Comment by marcellmueller** [comment]
> Looks good!
> 
> Unrelated to this, if we want to be less CPU intensive, we can set the archiver option `{ store: true }` to turn off compression

Cool, I will keep that in mind when we do some optimizations :+1:

**Pull request merged**